controllers/client/user.controller.js

```diff
@@ -27,3 +27,42 @@ module.exports.registerPost = async (req,res)=>{
     res.cookie("tokenUser" , user.tokenUser);
     res.redirect("/");
 }
+//[Get]/user/login
+module.exports.login = async(req, res)=>{
+    res.render("client/pages/user/login",{
+        pageTitle: "Trang Đăng Nhập"
+    })
+}
+//[Post]/user/loginPost
+module.exports.loginPost = async(req, res)=>{
+    const email = req.body.email;
+    const password = req.body.password;
+    const user = await User.findOne({
+        email: email,
+        deleted:false
+    });
+    if(!user){
+        req.flash("error" ,"Email Không Tồn Tại!");
+        res.redirect("back");
+        return;
+    }
```

Comment:
We should just return a generic message such as "Email and password combination not found".

If you specifically says "Email does not exist", an attacker can use this information to keep guessing account emails.